src/max31855/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 set (libname "max31855")
-set (libdescription "K type thermistor amplifier")
+set (libdescription "Thermocouple amplifier")
 set (module_src ${libname}.cxx)
```

Comment:
thermistor might be the more commonly accepted nomenclature for these types of devices.